src/sourcify-chains.ts

```diff
@@ -820,8 +827,8 @@ export function getSortedChainsArray(
       getPrimarySortKey(a) > getPrimarySortKey(b)
         ? 1
         : getPrimarySortKey(b) > getPrimarySortKey(a)
-        ? -1
-        : 0
+          ? -1
+          : 0
```

Comment:
Must be some problem with VS Code, because a I have not formatted the code, However, I have removed the additional white spaces, as you've asked. Additionally, I inform you that some tests will fail because the Symplexia Smart Chain is in maintenance. Thanks for you attention.

Reply:
Ok did you see my comment regarding the test files?

> I see you added new sources for the tests. Existing contracts sources didn't work for some reason? Just for context, I see you followed point 5.ii in [Sourcify's documentation](https://docs.sourcify.dev/docs/chain-support/), my question is: why points 5.i.a or 5.i.b or 5.i.c didn't work for you?